.github/workflows/ci.yml

```diff
@@ -85,8 +85,8 @@ jobs:
 
       - name: Activate virtualenv for later steps
         run: |
-          echo "::set-env name=VIRTUAL_ENV::$(pwd)/.venv"
-          echo "::add-path::$(pwd)/.venv/bin"
+          echo "name=VIRTUAL_ENV::$(pwd)/.venv" >> $GITHUB_ENV
```

Comment:
```suggestion
          echo "VIRTUAL_ENV=$(pwd)/.venv" >> $GITHUB_ENV
```

same below

Reply:
My bad, my latest force push was supposed to fix this.